ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-ocm-2.3.yaml

```diff
@@ -54,40 +53,25 @@ resources:
 tag_specification:
   name: "4.8"
   namespace: ocp
+test_binary_build_commands: |
+  go mod vendor
+  mkdir /.cache && chmod 775 -R /.cache && chmod 775 -R $GOPATH
 tests:
 - as: lint
-  commands: export GOLANGCI_LINT_CACHE=/tmp/golangci-cache make lint
+  commands: |
+    export GOLANGCI_LINT_CACHE=/tmp/.cache
+    export GOPROXY=https://proxy.golang.org
+    make lint
   container:
-    from: bin
+    from: test-bin
 - as: format-check
   commands: make format-check
   container:
-    from: bin
+    from: test-bin
 - as: unit-test
   commands: make REPORTS=${ARTIFACT_DIR} unit-test
   container:
-    from: bin
-- as: e2e-metal-assisted
-  steps:
-    cluster_profile: packet
-    dependencies:
-      ASSISTED_CONTROLLER_IMAGE: pipeline:assisted-installer-controller
-      ASSISTED_INSTALLER_IMAGE: pipeline:assisted-installer
-    workflow: baremetalds-assisted-install
```

Comment:
I'm not sure about having no e2e flow at all
Even though it's REST API, don't we want to make sure installer and controller are runnable in a full flow?

Reply:
You are right. Bad copy-pasta